test/gtest/ucs/test_log.cc

```diff
@@ -61,14 +63,21 @@ class log_test : public ucs::test {
         return system(cmd);
     }
 
+    std::string read_logfile() {
+        std::stringstream ss;
+        std::ifstream ifs(logfile);
```

Comment:
does it mean that it will rewrite `logfile` name and `unlink` will unsuccessful in the dtor?

Reply:
std::ifstream is only reading the file (before dtor)

Reply:
> std::ifstream is only reading the file (before dtor)

I see, thx for the explanation!